### ISSUE TITLE
Get familiar with Docusaurus

### ISSUE THREAD

**Issue by ipopescu** [casper-network/docs#320]
There are 2 repositories that make up the new docs: one is the app (the framework, the styles), and the second is the content.
https://github.com/casper-network/docs-app
https://github.com/casper-network/documentation

At the end, move the content from [here](https://github.com/casper-network/docs/pull/312) to docusaurus. Close the old PR and open a new PR.

Running Docusaurus in local host is pretty easy, You have to perform following few steps.

- Install node js
- Install yarn via npm: npm install --global yarn
- Clone two repos (https://github.com/casper-network/casper-node/docs-app and https://github.com/casper-network/casper-node/documentation) on your local machine
- You may mostly work on docs-app
- Go to docs-app and run following commands
- yarn install - required only once for a folder
- yarn run start - to start the local server (I have done remove '--no-open' part, so that it will automatically open the localhost page in our browser)
- Navigate to http://localhost:3000/
- More details on README.md at - https://github.com/casper-network/docs-app
